render-test-config.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
```

Comment:
We should probably look into merging this with `render_config.sh`, but that can happen later